### PR TITLE
Add private schemas

### PR DESCRIPTION
**Pull request by wpears**
Adds metadata for MA, MD, and MT. Depends on #153

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -900,13 +900,18 @@ test('Field tests', function(t){
...
 
   fs.readdirSync('test/data/fields')
     .filter(function(v){return v[0] !== '.' && v.indexOf('.') !== -1})
     .forEach(function(v){fieldFiles[path.basename(v, '.json')] = fs.readJsonSync(path.join('test/data/fields', v))});
 
   data.forEach(function(source){
+
+    //Non-public
+    if(!url.parse(source.url).hostname) return;
+    count+=2;
```
Comment (hkeeler) [inline test/test.js]: This `count` doesn't seem to be used for anything?

Comment (wpears) [inline test/test.js]: Well-spotted; was a holdover from structuring that test in a certain way; but I decided it was okay to just explicitly `end()` it without doing any jiggery pokery

**Comment by hkeeler** [comment]
Travis is sad again: 
* https://travis-ci.org/cfpb/grasshopper-loader/builds/97742416#L202

_This_ time if couldn't build the `loader` image because some Ubuntu packages couldn't be downloaded.  I'll kick it back off...

**Comment by hkeeler** [comment]
Huh.  Looks like when you do a _Restart Build_ in Travis, it uses the same URL, so you can link to where it broke.  That's not nice.

**Comment by hkeeler** [comment]
Travis = :grinning:

**Pull request merged**